diff -p -up gnome-themes-extras-2.22.0/gtk-themes/Darklooks/gtk-2.0/gtkrc~ gnome-themes-extras-2.22.0/gtk-themes/Darklooks/gtk-2.0/gtkrc
--- gnome-themes-extras-2.22.0/gtk-themes/Darklooks/gtk-2.0/gtkrc~	2008-03-09 13:38:00.000000000 +0100
+++ gnome-themes-extras-2.22.0/gtk-themes/Darklooks/gtk-2.0/gtkrc	2008-05-25 23:24:35.000000000 +0200
@@ -1,6 +1,6 @@
 # Set GtkSettings color scheme property.
 # This can be overriden (via an xsetting) with eg. the gnome-appearance-properties.
-gtk_color_scheme = "fg_color:#E6E6E6\nbg_color:#555753\nbase_color:#2E3436\ntext_color:#D3D7CF\nselected_bg_color:#3F403D\nselected_fg_color:#7AA3CC\ntooltips_bg_color:#EDDE5C\ntooltips_fg_color:#000000"
+gtk_color_scheme = "fg_color:#E6E6E6\nbg_color:#555753\nbase_color:#2E3436\ntext_color:#D3D7CF\nselected_bg_color:#3F403D\nselected_fg_color:#7AA3CC\ntooltip_bg_color:#EDDE5C\ntooltip_fg_color:#000000"
 
 style "clearlooks-default"
 {
